northstar/src/runtime/minijail.rs

```diff
@@ -685,6 +706,7 @@ impl ProcessSync {
         // is owned by the parent.
         let (childfd, parentfd) =
             pipe().map_err(|e| Error::Os("Failed to create pipe".to_string(), e))?;
+        error!("opening pipe ({}, {})", childfd, parentfd);
```

Comment:
Logs start with a capital.

Is this an `error!`?

Reply:
whoops this was for debugging